recipe/meta.yaml

```diff
@@ -1,11 +1,11 @@
-{% set version = "2021.13.0" %}
```

Comment:
Please note that `2022.0.0.rc1` is not a final tag for 2022.0.0 release. Is it really crucial to merge this right now or it can wait, let's say, until the beginning of the next week?
FYI @omalyshe

Reply:
It is not a build dependency, but runtime dependency for the intel's repack. I would prefer to unblock several compiler PRs now:

https://github.com/conda-forge/intel-compiler-repack-feedstock/pull/52
https://github.com/conda-forge/intel_repack-feedstock/pull/82
https://github.com/conda-forge/pti-gpu-unitrace-feedstock/pull/1

So all the packages that depends on new compiler can be unblocked.

Reply:
oneTBB 2022.0.0 has been released - https://github.com/oneapi-src/oneTBB/releases/tag/v2022.0.0